monai/utils/type_conversion.py

```diff
@@ -99,11 +98,15 @@ def get_dtype(data: Any):
 
 
 def convert_to_tensor(
-    data, dtype: Optional[torch.dtype] = None, device: Optional[torch.device] = None, wrap_sequence: bool = False
+    data,
+    dtype: Optional[torch.dtype] = None,
+    device: Optional[torch.device] = None,
+    wrap_sequence: bool = False,
+    track_meta: bool = False,
 ):
     """
-    Utility to convert the input data to a PyTorch Tensor. If passing a dictionary, list or tuple,
-    recursively check every item and convert it to PyTorch Tensor.
+    Utility to convert the input data to a PyTorch Tensor, if tracking meta, convert to `MetaTensor`.
+    If passing a dictionary, list or tuple, recursively check every item and convert it to PyTorch Tensor.
```

Comment:
perhaps adding "If `track_meta` is `True`, the output will be a MetaTensor; otherwise, the output will be a regular torch Tensor."

Reply:
Updated in the latest commit.

Thanks.